1.9.0

```diff
@@ -258,6 +258,40 @@
           }
         }
       ]
+    },
+    {
+      "sobjectName":"Campaign",
+      "instructionTypeName":"mdTaskListView",
+      "taskList":[
+        {
+          "jiraTicket": "AC-59235",
+          "description": "Delete AllActiveCampaigns List View",
+          "actionType": "deleteListView",
+          "optionsMap": {
+            "fullName": "Campaign.AllActiveCampaigns",
+            "filterScope": "Everything",
+            "operation": "delete"
+          }
+        }
+      ]
+    },
+    {
+      "sobjectName":"Campaign",
+      "instructionTypeName":"mdTaskListView",
+      "taskList":[
+        {
+          "jiraTicket": "AC-59235",
+          "description": "Delete AllActiveCampaigns List View",
+          "actionType": "deleteListView",
+          "optionsMap": {
+            "fullName": "Campaign.AllActiveCampaigns",
```

Comment:
use actionTypeName instead of fullName like mentioned above. There are definitions in the mdInstructionWrapper class that explain the available variables.

Reply:
Done